src/dirtyfields/compat.py

```diff
@@ -13,6 +14,11 @@ def is_db_expression(value):
         return isinstance(value, (BaseExpression, Combinable))
 
 
+def is_deferred(model, field):
```

Comment:
Yes I thought it too :) . I can create u `utils.py` with this function if you like.